frontend/src/components/Dashboard/ScoresTableSection/BodyFields.jsx

```diff
@@ -1,37 +0,0 @@
-import React from 'react'
-import {connect} from 'react-redux'
-import {TableBody, TableRow, TableRowColumn} from 'material-ui'
-
-
-class BodyFields extends React.Component {
```

Comment:
I think BodyFields is not appropriate name. Field has a little bit different meaning. As for me it woud be better to use BodySection